corehq/apps/hqwebapp/templates/hqwebapp/crispy/field/field_with_text.html

```diff
@@ -9,7 +9,6 @@
         </label>
         <div class="controls" style="padding-top: 5px;">
             <input type="hidden"
-                   name="{{ field.html_name }}"
```

Comment:
> @orangejenny Is this related to #9395 ?

I don't think so...but I do agree with Ethan that this seems like it's going to prevent the value from getting saved.